### PR TITLE
Adding links to contributor's github profiles

### PR DESCRIPTION
**Pull request by mrchief**
Added what I could find. Couldn't find remaining ones.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,12 +168,12 @@ var envprops = require('envprops'),
...
 ## Contributors:
 
-- Yuriy Nemtsov
+- [Yuriy Nemtsov](https://github.com/nemtsov)
 - Lee Cookson
-- Dominique Boucher
-- Hrusikesh Panda
+- [Dominique Boucher](https://github.com/Solognot)
+- [Hrusikesh Panda](https://github.com/mrchief)
 - Doug Reiter
```
Comment (nemtsov) [inline README.md]: https://github.com/ddrinf

File: README.md [hunk 2]
```diff
@@ -168,12 +168,12 @@ var envprops = require('envprops'),
...
-- Christopher Montgomery
+- [Christopher Montgomery](https://github.com/chmontgomery)
 - Robert DiNicolas
```
Comment (nemtsov) [inline README.md]: https://github.com/bstar

File: README.md [hunk 3]
```diff
@@ -168,12 +168,12 @@ var envprops = require('envprops'),
 
 ## Contributors:
 
-- Yuriy Nemtsov
+- [Yuriy Nemtsov](https://github.com/nemtsov)
 - Lee Cookson
```
Comment (nemtsov) [inline README.md]: https://github.com/leecookson

**Comment by mrchief** [comment]
@nemtsov Is this good to go then?

**Pull request merged**